test/plugin.test.js

```diff
@@ -525,3 +627,28 @@ QUnit.test('removeSession removes sessions', function(assert) {
   removeSession(sessions, initData2);
   assert.deepEqual(sessions, [], 'does nothing when no sessions');
 });
+
+QUnit.test('emeError properly handles various parameter types', function(assert) {
+  let errorObj;
+  const player = {
+    tech_: {
+      el_: videojs.EventTarget()
+    },
+    error: (obj) => {
+      errorObj = obj;
+    }
+  };
+  const emeError = emeErrorHandler(player);
+
+  emeError(undefined);
+  assert.equal(errorObj.message, undefined, 'undefined error message');
```

Comment:
For good measure, might want to use `typeof errorObj.message` and `'undefined'` instead of referencing `undefined` directly.